.bash_prompt

```diff
@@ -164,7 +164,15 @@ sexy_bash_prompt_branch_exists () {
 sexy_bash_prompt_parse_git_ahead () {
   # Grab the local and remote branch
   branch="$(sexy_bash_prompt_get_git_branch)"
-  remote_branch="origin/$branch"
+  
+  #remote_branch="origin/$branch"
+  #Compute the remote tracking branch, if no remote branch then echo 0
+  branchnamefull=$(git symbolic-ref -q HEAD);
+  remote_branch=$(git for-each-ref --format='%(upstream:short)' $branchnamefull)
+  if [[  "$remote_branch" == "" ]]
```

Comment:
Instead of bailing out when `remote_branch` is empty, we should fallback to `origin/$branch`. It's possible that the user didn't properly set up `upstream` tracking (I get this on a lot of my repos, not sure how/why):

```bash
remote_branch="$(git rev-parse --symbolic-full-name "@{upstream}" 2> /dev/null || echo -n "origin/$branch")"
```